### PR TITLE
Fix DMS requirement number

### PR DESCRIPTION
**Pull request by nden**
Resolves [RCAL-nnnn](https://jira.stsci.edu/browse/RCAL-nnnn)

Closes #

This PR addresses ...

**Checklist**
- [ ] added entry in `CHANGES.rst` under the corresponding subsection
- [ ] updated relevant tests
- [ ] updated relevant documentation
- [ ] updated relevant milestone(s)
- [ ] added relevant label(s)
- [ ] ran regression tests, post a link to the Jenkins job below. [How to run regression tests on a PR](https://github.com/spacetelescope/romancal/wiki/Running-Regression-Tests-Against-PR-Branches)

### CODE REVIEW

File: romancal/regtest/test_wfi_dq_init.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def test_dq_init_image_step(rtdata, ignore_asdf_paths):
     assert diff.identical, diff.report()
 
 
-@metrics_logger("DMS26")
+@metrics_logger("DMS25")
 @pytest.mark.bigdata
 def test_dq_init_grism_step(rtdata, ignore_asdf_paths):
     """DMS26 Test: Testing retrieval of best ref file for grism data,
```
Comment (schlafly) [inline romancal/regtest/test_wfi_dq_init.py:74]:
```suggestion
    """DMS25 Test: Testing retrieval of best ref file for grism data,
```

**Review by ddavis-stsci — APPROVED** [review]

**Review by schlafly — APPROVED** [review]
Thanks Nadia.  I think if you Ctrl-F DMS26 -> replace DMS25 you'll hit five more instances we should probably get, though presumably this is the only one they "really" care about.

**Comment by nden** [comment]
Oo, I never checked. Fixed all of them now.

**Review by schlafly — APPROVED** [review]
Thanks!

**Pull request merged**